docs/auditscan/scan/help.go

```diff
@@ -0,0 +1,11 @@
+package scan
+
+const Description = "Scan files located on the local file-system with Xray."
+
+var Usage = []string{"jfrog scan [command options] <source pattern> ",
```

Comment:
I think we should have a const in the code, that will be initialized to either "jfrog" or "jf" according to the executable name. We can then use this constant in all help pages, such as the above.